server/game/cards/01-DTR/SunInYerEyes.js

```diff
@@ -8,7 +8,8 @@ class SunInYerEyes extends ActionCard {
             target: {
                 activePromptTitle: 'Select a dude who has sun in their eyes',
                 cardCondition: { location: 'play area', participating: true, controller: 'opponent' },
-                cardType: 'dude'
+                cardType: ['dude'],
+                gameAction: ['decreaseBullets', 'setAsDraw']
```

Comment:
Not sure about the others where there is just one action, but this is not correct as it would prevent the whole effect if there is only one of the actions prevented.

Reply:
Surprising; it was [The Mayor's Office](https://github.com/townteki/townsquare/blob/master/server/game/cards/02.3-EDS/TheMayorsOffice.js#L12) that convinced me it worked the other way.